src/ui/auth/authenticator.go

```diff
@@ -37,22 +37,26 @@ const (
 
 var lock = NewUserLock(frozenTime)
 
-// Authenticator provides interface to authenticate user credentials.
-type Authenticator interface {
+// AuthenticateHelper provides interface to authenticate user credentials.
+type AuthenticateHelper interface {
 
 	// Authenticate ...
 	Authenticate(m models.AuthModel) (*models.User, error)
+	// Create user in local database
```

Comment:
We should explicitly document that if the user exists in DB the ID of the user entry will be updated to the parameter.